### ISSUE TITLE
missing dependencies not installed

### ISSUE THREAD

**Issue by robvanmieghem** [threefoldtech/jumpscale_lib#77]
#### Installation information

- jumpscale version: v9.0.0-1632-g39ede236
- operating system: Ubuntu 18.04.1 LTS

#### Detailed description

j.clients.rivine. does not work due to missing ed25519 and pyblake2 dependencies

#### Steps to reproduce
```
export JUMPSCALEBRANCH=development
curl https://raw.githubusercontent.com/threefoldtech/jumpscale_core/$JUMPSCALEBRANCH/install.sh?$RANDOM > /tmp/install_jumpscale.sh;bash /tmp/install_jumpscale.sh
```
### workaround:
```
pip3 install ed25519
pip3 install pyblake2
```

**Comment by zaibon** [comment]
as far as I know when you install jumpscale in dev mode this is the expected behavior

**Comment by ahussein** [comment]
How did you install jumpscale ?
did you set the JSFULL=1  environment variable before installing ?

**Comment by robvanmieghem** [comment]
I used 
```
export JUMPSCALEBRANCH={BRANCH}
curl https://raw.githubusercontent.com/threefoldtech/jumpscale_core/$JUMPSCALEBRANCH/install.sh?$RANDOM > /tmp/install_jumpscale.sh;bash /tmp/install_jumpscale.sh
```

**Comment by ahussein** [comment]
then its expected behavior. Libs are installed without dependencies by default and if you need to have the dependencies then you will need to either install the specific deps manually when needed or set the JSFULL=1 environment variable before installing to install all the deps.

**Comment by robvanmieghem** [comment]
I understand, the thing is also I did not know the dependencies
`j.clients.rivine.[TAB]` did nothing. I don't know how someone would be able to know the dependencies this way, Is it common behaviour to document them or is there another way to see what is needed?

**Comment by ahussein** [comment]
Yes, right now there is no way for the developer to know what kind of deps they need to work with their client except for checking the error.
I will update the docs to reflect that.

**Comment by ahussein** [comment]
fixed by 39a7505
